docs/user/ai_model_serving.md

```diff
@@ -0,0 +1,517 @@
+1. Add the actual image reference to the `image:` parameter field value
+1. Create the object using the file or make it part of a manifest (kustomization)
+
+The following example shows the process of reusing `microshift-ai-model-serving` manifest's files
+to re-create OVMS `ServingRuntime` in the workload's namespace:
+```sh
+# Get image reference for the 'ovms-image'
+OVMS_IMAGE="$(jq -r '.images | with_entries(select(.key == "ovms-image")) | .[]' /usr/share/microshift/release/release-ai-model-serving-"$(uname -i)".json)"
+
+# Duplicate the original ServingRuntime yaml
+cp /usr/lib/microshift/manifests.d/001-microshift-ai-model-serving/runtimes/ovms-kserve.yaml ./ovms-kserve.yaml
+
+# Update the image reference
+sed -i "s,image: ovms-image,image: ${OVMS_IMAGE}," ./ovms-kserve.yaml
+```
+
+Then you can re-create the ServingRuntime in a custom namespace:
+```sh
+oc create -n ai-demo -f ./ovms-kserve.yaml
+```
+
+Alternatively, if the preceding procedure is part of a bootc Containerfile and
+the ServingRuntime ends up as part of new manifest, the namespace can set in the `kustomization.yaml`:
+```yaml
+namespace: ai-demo
+```
+
+### Creating the InferenceService custom resource
+
+Next, we need to create InferenceService custom resource (CR).
+InferenceService instructs kserve on how to create a Deployment for serving the model.
+Kserve uses the ServingRuntime based on `modelFormat` specified in InferenceService.
+
+It's possible to add extra arguments that will be passed to the model server using `.spec.predictor.model.args`.
+
+The following is an example of an InferenceService with a model in the `openvino_ir` format.
+It features an additional argument, `--layout=NHWC:NCHW` to make OVMS accept the
+request input data in a different layout than the model was originally exported with.
+Extra args are passed through to the OVMS container.
+
+For more information about the InferenceService CR refer to
+[RHOAI documentation](https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.17/html/serving_models/serving-large-models_serving-large-models#inferenceservice).
+
+Example `InferenceService` object with an `openvino_ir` model format
+```yaml
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: ovms-resnet50
+spec:
+  predictor:
+    model:
+      protocolVersion: v2
+      modelFormat:
+        name: openvino_ir
+      storageUri: "oci://localhost/ovms-resnet50:test"
+      args:
+      - --layout=NHWC:NCHW
+```
+
+Save the the InferenceService example to a file, then create it on the cluster:
+```sh
+oc create -n ai-demo -f ./FILE.yaml
+```
+
+Output:
+```
+inferenceservice.serving.kserve.io/ovms-resnet50 created
+```
+
+Soon, a Deployment and a Pod should appear in that namespace.
+Depending on the size of ServingRuntime's image and the size of the ModelCar OCI image,
+it may take a while for the Pod to become ready.
+
+You can check the status of the Deployment with following command:
```

Comment:
I think we're too nice here :). These are pure kube commands, so let's try to shorten it so that we only include info relevant to the current topic.